sql/test_user.sql

```diff
@@ -31,13 +31,17 @@ CREATE TABLE `user` (
   `USERNAME` varchar(30) NOT NULL,
   `PASSWORD` varchar(15) DEFAULT NULL,
   `ROLEID` int(20) DEFAULT NULL,
+
+  `userstatus` varchar(20) DEFAULT NULL,
+
```

Comment:
In the first case you can remove that .I forgot to remove it.

In the second case where did you find 2 columns ? which columns u are
talking about?

On Tue, Oct 6, 2015 at 10:30 AM, Nagarjuna Myla <notifications@github.com>
wrote:

> In sql/test_user.sql
> <https://github.com/chasethenag420/sbs/pull/22#discussion_r41295318>:
>
> > @@ -31,13 +31,17 @@ CREATE TABLE `user` (
> >    `USERNAME` varchar(30) NOT NULL,
> >    `PASSWORD` varchar(15) DEFAULT NULL,
> >    `ROLEID` int(20) DEFAULT NULL,
> > +
> > +  `userstatus` varchar(20) DEFAULT NULL,
> > +
>
> Why two columns with same name an different length
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chasethenag420/sbs/pull/22/files#r41295318>.
>